### PR TITLE
chore: return to_date result as timestamps

### PR DESCRIPTION
**Pull request by raphaelvignes**
* Make `to_date` return timestamps.

### CODE REVIEW

File: server/src/weaverbird/backends/pypika_translator/translators/base.py [hunk 1]
```diff
@@ -1262,11 +1263,13 @@ def todate(
                     convert_fn = StrToDate
                 case ToDateOp.PARSE_DATE:
                     convert_fn = ParseDate
+                case ToDateOp.TO_TIMESTAMP:
+                    convert_fn = functions.Timestamp
                 case _:
                     raise NotImplementedError(f"[{self.DIALECT}] todate has no set operator")
             date_selection = convert_fn(col_field, step.format)
         else:
```
Comment (lukapeschke) [inline server/src/weaverbird/backends/pypika_translator/translators/base.py:1271]: We're still returning something that's not a timestamp if the format is specified. Could you please add `self._cast_to_timestamp` of the result in both cases ?

File: server/src/weaverbird/backends/pypika_translator/translators/postgresql.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ class PostgreSQLTranslator(SQLTranslator):
         integer="INTEGER",
         text="TEXT",
         datetime="TIMESTAMP",
+        timestamp="TIMESTAMP",
```
Comment (lukapeschke) [inline server/src/weaverbird/backends/pypika_translator/translators/postgresql.py:33]: Could you please set the `TO_DATE_OP` to `TO_TIMESTAMP` ?

**Review by lukapeschke — CHANGES_REQUESTED** [review]
Thanks for the proposal! I've got a few remarks and suggestions, see inline comments

File: server/src/weaverbird/backends/pypika_translator/operators.py [hunk 3]
```diff
@@ -20,3 +20,4 @@ class ToDateOp(Enum):
     TO_DATE = auto()
```
[the remark this answers is not in the document]
Comment (raphaelvignes) [reply server/src/weaverbird/backends/pypika_translator/operators.py:20]: :+1:

File: server/src/weaverbird/backends/pypika_translator/translators/postgresql.py (hunk 2 above)
Comment (raphaelvignes) [reply server/src/weaverbird/backends/pypika_translator/translators/postgresql.py:33]: good catch.

**Review by jGundermann — APPROVED** [review]
LGTM

**Pull request merged**